_sections/about-me.html

```diff
@@ -14,9 +14,9 @@
   <script type="text/javascript" async src="//cdn.credly.com/assets/utilities/embed.js"></script>
 </div>
 
-<p>I've also obtained my Basic Chef Fluency and Local Cookbook Development badges</p>
+<p>I've also worked with Chef and Ansible, as well as terraform</p>
```

Comment:
Nit: Suggest making the casing consistent between the tools. i.e. either capitalize Terraform or use lower case chef and ansible

Reply:
Done